cf/timeduration.py

```diff
@@ -1162,12 +1163,16 @@ def equals(
         if self is other:
             return True
 
+        message = is_log_level_info(logger)
```

Comment:
Can we call `message` instead `info` as we have done in other modules and/or methods, for consistency? Just so it is more transparent to us in future.

Reply:
Done: bcbe0d073b167b0fd61e9b6876e54168df6e13ad